.circleci/config.yml

```diff
@@ -103,6 +103,12 @@ jobs:
           key: v1-dependency-cache-py27-{{ checksum "requirements.txt" }}
       - attach_workspace:
           at: /
+      - run:
+          name: Clear apt cache
+          command: sudo rm -rf /var/cache/apt/archives && sudo ln -s ~/.apt-cache /var/cache/apt/archives && mkdir -p ~/.apt-cache/partial
```

Comment:
I think apt-get update in the next step should do the trick, so I'm wondering why we're doing this cleanup here?

Reply:
looking at it, we're deleting the cache in /var/cache/apt and creating a symlink to ~/.apt-cache so that we can then cache ~/.apt-cache in CircleCI build cache.

CircleCI doesn't support caching things outside of /home

So, this cleanup is necessary so that we can create our symlink and use the CircleCI apt cache